src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -44,9 +46,12 @@ public function configureOptions(OptionsResolver $resolver)
                     return null;
                 }
 
+                $this->locale = $options['locale'];
```

Comment:
agreed. It confused me before: it was so hard find place where should I save locale from options 🤣